class-udm-updater.php

```diff
@@ -95,6 +100,34 @@ public function __construct($mothership, $muid, $relative_plugin_file, $options
 		if (version_compare($wp_version, '5.5', '<')) {
 			add_filter('auto_update_plugin', array($this, 'auto_update_plugin'), 20, 2);
 		}
+
+		add_action('admin_init', array($this, 'disconnect_cloned_site'));
+	}
+
+	/**
+	 * Disconnect a cloned site by unsetting the `email` option and resetting PUC update cache and at the same time adding `site_host_path` as a plugin option for a cloned site checking purpose in future
+	 */
+	public function disconnect_cloned_site() {
+
+		global $pagenow; 
+
+		if (defined('DOING_AJAX') && DOING_AJAX || 'plugins.php' !== $pagenow) return;
```

Comment:
@DavidAnderson684 Aah.. right, thanks for spotting that one. Looking at the code there is another page that displays the UI, which is `update-core.php`. I've updated the code, but rather than adding `update-core.php`, I removed the line and let the `disconnect_cloned_site` method be called via two methods associated with the page hooks. https://github.com/DavidAnderson684/simba-plugin-manager-updater/pull/11/commits/767721dade31aa485238f19fd7dcfa620396a847